Calibration/TkAlCaRecoProducers/python/AlcaSiStripGainsAAGHarvester_cff.py

```diff
@@ -9,9 +9,4 @@
 
 DQMStore = cms.Service("DQMStore")
 
-from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
```

Comment:
@schneiml why is this being removed?
It has been extremely useful in the past to be able to know the run number when checking the results of the PCL in DQM GUI. Also as far as I can see there are other instances of `DQMEventInfo` in the Harvesting step of other PCL workflows: https://github.com/cms-sw/cmssw/search?l=Python&q=DQMEventInfo
Please clarify or provide a suitable replacement.

Reply:
You cannot run `DQMEDAnalyzer`s in harvesting. An equivalent `DQMEDHarvester` could be built, many of the things in `DQMEventInfo` don't make sense in harvesting anyways.

Re the other usages: It is technically save to load an run the module, but it will not produce any output. This has worked int the past by accident, but would silently produce incorrect results e.g. in multi-run harvesting. I removed it here after it's output disappeared in the PR comparison, to clarify that the module is no longer useful here.

Reply:
> An equivalent DQMEDHarvester could be built, many of the things in DQMEventInfo don't make sense in harvesting anyways.

can you please provide such implementation?

> Re the other usages: It is technically save to load an run the module, but it will not produce any output. This has worked int the past by accident, but would silently produce incorrect results e.g. in multi-run harvesting. I removed it here after it's output disappeared in the PR comparison, to clarify that the module is no longer useful here.

if it is so, then I think must be removed everywhere else...

Reply:
Ok, that should be separate PR, potentially to be merged before. I can give it a try tomorrow.

Reply:
thanks!